app.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+var productList = [];
+var leftImage = document.getElementById('left-image');
+var rightImage = document.getElementById('right-image');
+var midImage = document.getElementById('mid-image');
+var numClicks = 0;
```

Comment:
totalClicks might have been a slightly more clear name for this var.  numClicks doesn't reveal what it is the number of clicks of.